verifier/testreport.py

```diff
@@ -806,32 +809,39 @@ def add_to_results_by_key(self, label, results, input_data, test, key_list):
                 except:
                     pass
 
-    def check_simple_text_diffs(self, test_list, category):
+    def check_text_diffs(self, test_list, category):
         results = defaultdict(list)
         all_checks = ['insert', 'delete', 'insert_digit', 'insert_space', 'delete_digit',
                       'delete_space', 'replace_digit', 'replace_dff', 'replace_diff', 'whitespace_diff',
                       'replace', 'diff_in_()', 'parens', '() --> []', '[] --> ()',
-                      'comma_type', 'unexpected_comma']
-
+                      'comma_type', 'unexpected_comma', 'result_type_difference', 'boolean_difference',
+                      'error_in_key']
```

Comment:
Should we add `other_list_differences` here, too?

Reply:
I see that it's done, and you correctly put it in the singular, despite my comment above putting it in the plural.